tooling/create-new-effect.mjs

```diff
@@ -1,49 +1,67 @@
 import fs from "fs";
 import path from "path";
-import {Command} from "commander";
-import sanitize from "sanitize-filename";
-import {getFoundryId, isValidFoundryImage} from "./utils.js";
-import {DIR_PACKS_SOURCE, MACRO_AUTHOR} from "./consts.mjs";
+import {Command, Option} from "commander";
```

Comment:
This is updated to generate files in the new folder structure. No other functional changes.